asf_tools/hand/calculate.py

```diff
@@ -158,11 +158,10 @@ def main():
         epilog='For watershed boundaries, see: https://www.hydrosheds.org/page/hydrobasins',
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    parser.add_argument('out_raster', type=Path,
-                        help='HAND GeoTIFF to create')
-    parser.add_argument('vector_file', type=Path,
-                        help='Vector file of watershed boundary (hydrobasin) polygons to calculate HAND over. '
-                             'Vector file Must be openable by GDAL, see: https://gdal.org/drivers/vector/index.html')
+    parser.add_argument('out_raster', help='HAND GeoTIFF to create')
```

Comment:
writing to /vsicurl generally won't work, and I don't *think* you can write a COG to /vsis3 , so we might consider keeping `out_raster` as a Path on the local file system?

Reply:
Technically, `/vsis3` allows writing of files to s3:
https://gdal.org/user/virtual_file_systems.html#vsis3-aws-s3-files

Which is why I made the change, but I def. haven't tested it